builder/proxmox/common/config.go

```diff
@@ -171,6 +172,59 @@ type vgaConfig struct {
 	Memory int    `mapstructure:"memory"`
 }
 
+// Allows passing through a host PCI device into the VM. For example, a graphics card
+// or a network adapter. Devices that are mapped into a guest VM are no longer available
+// on the host. A minimal configuration only requires either the `host` or the `mapping`
+// key to be specifed.
+//
+// Note: VMs with passed-through devices cannot be migrated.
+//
+// Example configuration (HCL):
+//
+// ```hcl
+// pci_devices {
+// host          = "0000:0d:00.1"
```

Comment:
In HCL2 generally, the contents of a block are intented (with 2 spaces) relative to their immediate parent, I suggest we do the same here, is this OK?

Reply:
Sorry, I can't get my editor to do that. If I add two spaces it just reindents everything extremely weirdly [like the rng0](https://github.com/romantomjak/packer-plugin-proxmox/blob/edcd09a5834cb994c848f6145d68b4573a2ade10/builder/proxmox/common/config.go#L130-L138)

Reply:
It's OK! I can do this change in a subsequent PR either way, so I think I'll merge your PR if everything checks out, and fix this later :)